### ISSUE TITLE
Vuido, do you support the picture now?

### ISSUE THREAD

**Issue by wxbing1207** [mimecorg/vuido#45]
Vuido, do you support the picture now?

**Comment by mimecorg** [comment]
No, not yet. Only the features listed in the documentation are supported.

**Comment by chuanhaisoft** [comment]
https://github.com/andlabs/libui/search?q=uiImage&unscoped_q=uiImage

**Comment by chuanhaisoft** [comment]
https://github.com/andlabs/libui/blob/master/windows/image.cpp
